lightning/src/ln/functional_tests.rs

```diff
@@ -8422,36 +8424,94 @@ fn test_manually_accept_inbound_channel_request() {
 
-	let close_msg_ev = nodes[1].node.get_and_clear_pending_msg_events();
-	assert_eq!(close_msg_ev.len(), 1);
+	// Continue channel opening process until channel update messages are sent.
+	nodes[0].node.handle_accept_channel(nodes[1].node.get_our_node_id(), &accept_channel);
+	let (temporary_channel_id, tx, funding_outpoint) = create_funding_transaction(&nodes[0], &nodes[1].node.get_our_node_id(), 100_000, 42);
+	nodes[0].node.unsafe_manual_funding_transaction_generated(temporary_channel_id, nodes[1].node.get_our_node_id(), funding_outpoint).unwrap();
+	check_added_monitors!(nodes[0], 0);
 
-	let events = nodes[1].node.get_and_clear_pending_events();
-	match events[0] {
-		Event::ChannelClosed { user_channel_id, .. } => {
-			assert_eq!(user_channel_id, 23);
-		}
+	let funding_created = get_event_msg!(nodes[0], MessageSendEvent::SendFundingCreated, nodes[1].node.get_our_node_id());
+	nodes[1].node.handle_funding_created(nodes[0].node.get_our_node_id(), &funding_created);
+	check_added_monitors!(nodes[1], 1);
+	expect_channel_pending_event(&nodes[1], &nodes[0].node.get_our_node_id());
+
+	let funding_signed = get_event_msg!(nodes[1], MessageSendEvent::SendFundingSigned, nodes[0].node.get_our_node_id());
+	nodes[0].node.handle_funding_signed(nodes[1].node.get_our_node_id(), &funding_signed);
+	check_added_monitors!(nodes[0], 1);
+	let events = &nodes[0].node.get_and_clear_pending_events();
+	assert_eq!(events.len(), 2);
+	match &events[0] {
+		crate::events::Event::FundingTxBroadcastSafe { funding_txo, .. } => {
+			assert_eq!(funding_txo.txid, funding_outpoint.txid);
+			assert_eq!(funding_txo.vout, funding_outpoint.index.into());
+		},
 		_ => panic!("Unexpected event"),
-	}
+	};
+	match &events[1] {
+		crate::events::Event::ChannelPending { counterparty_node_id, .. } => {
+			assert_eq!(*&nodes[1].node.get_our_node_id(), *counterparty_node_id);
+		},
+		_ => panic!("Unexpected event"),
+	};
+
+	mine_transaction(&nodes[0], &tx);
+	mine_transaction(&nodes[1], &tx);
+
+	let as_channel_ready = get_event_msg!(nodes[1], MessageSendEvent::SendChannelReady, nodes[0].node.get_our_node_id());
+	nodes[1].node.handle_channel_ready(nodes[0].node.get_our_node_id(), &as_channel_ready);
+	let as_channel_ready = get_event_msg!(nodes[0], MessageSendEvent::SendChannelReady, nodes[1].node.get_our_node_id());
+	nodes[0].node.handle_channel_ready(nodes[1].node.get_our_node_id(), &as_channel_ready);
+
+	expect_channel_ready_event(&nodes[0], &nodes[1].node.get_our_node_id());
+	expect_channel_ready_event(&nodes[1], &nodes[0].node.get_our_node_id());
+
+	// Assert that the overriden base fee surfaces in the channel update.
+	let channel_update = get_event_msg!(nodes[1], MessageSendEvent::SendChannelUpdate, nodes[0].node.get_our_node_id());
+	assert_eq!(channel_update.contents.fee_base_msat, 555);
+
+	get_event_msg!(nodes[0], MessageSendEvent::SendChannelUpdate, nodes[1].node.get_our_node_id());
```

Comment:
I removed the force close steps that were present previously. Seems this is enough testing for manual acceptance?